templates/voms.json.j2

```diff
@@ -1,8 +1,10 @@
 {
+{% if caso_vos %}
```

Comment:
if you rebased these changes should not appear here :(

Reply:
I thought that was your change... Sorry man, rebases are my kryptonite.